File: testdata/jennies/builders/dashboard_panel/JavaBuilder/dashboard/PanelBuilder.java
package dashboard;


public class PanelBuilder implements cog.Builder<Panel> {
    protected final Panel internal;
    
    public PanelBuilder() {
        this.internal = new Panel();
    }
    public PanelBuilder onlyFromThisDashboard(Boolean onlyFromThisDashboard) {
        this.internal.onlyFromThisDashboard = onlyFromThisDashboard;
        return this;
    }
    public Panel build() {
        return this.internal;
    }
}

File: testdata/jennies/builders/dashboard_panel/CSharpBuilder/src/Grafana/Foundation/Dashboard/PanelBuilder.cs
namespace Grafana.Foundation.Dashboard;


public class PanelBuilder : Cog.IBuilder<Panel>
{
    protected readonly Panel @internal;

    public PanelBuilder()
    {
        this.@internal = new Panel();
    }

    public PanelBuilder OnlyFromThisDashboard(bool onlyFromThisDashboard)
    {
        this.@internal.OnlyFromThisDashboard = onlyFromThisDashboard;
        return this;
    }

    public Panel Build()
    {
        return this.@internal;
    }
}
